features/dlg_crash_info/README.md

```diff
@@ -0,0 +1,253 @@
+
+```c
+typedef __PACKED_STRUCT
+{
+	uint8_t 					data_valid;
+	uint32_t					epoch;
+	dci_last_fault_hdlr_t		fault_handler;
+	cortex_m0_stack_frame_t		stack_frame;
+	uint8_t						num_of_call_vals;
+	uint32_t					call_trace[DCI_CALL_DEPTH];
+	
+}dci_fault_info_t;
+```
+
+- data_valid:  Sanity check for valid data.
+- epoch:  Implemented by the user for optional time tracking.  
+- fault_handler:  as described above for dci_last_fault_hdlr_t
+- stack_frame:  Stack frame that is pushed by the Cortex-M0+ device, on exception entry, as specified by the [exception entry model](https://developer.arm.com/documentation/dui0662/b/The-Cortex-M0--Processor/Exception-model/Exception-entry-and-return)
+
+    ![Stack Frame](assets/stack_frame.PNG)
+- num_of_call_vals:  Number of linked addresses found on the stack.  Used to determine valid number of values in call_trace.
+- call_trace:  Call stack prior to exception entry.  Memory values determined as Linked Addresses on the stack.  
+> __Note:__ 
+The DCI Module uses the property of the ARM core pushing linked addresses to the stack as odd values.  The module looks for values on the stack that are odd, and within executable range.  It does not decipher between code and data, so it is possible that data could be interpreted as linked.  It happens rarely, as this requires an odd value, within executable range (RAM Heap or data) to be pushed to the stack by the assembler. 
+
+## DCI Service and BLE Usage
+
+### *Advertising*
+
+This example uses advertising to indicate to a central that there is reset data available.  The advertising payload uses service data, and then appends the number of resets from dci_data_t.  This allows for a central to scan and determine if a device has reset information it can connect and extract data from.  Please see the following example for more references on ![service data adertising](https://github.com/dialog-semiconductor/BLE_SDK6_examples/tree/main/connectivity/svc_data_beacon)
+
+This example uses the UUID from the DCI Service and appends a uint8_t for the number of resets.
+
+***Advertising Data***
+
+| AD # | AD Length | AD Type                          | AD Data                                               |
+|------|-----------|----------------------------------|-------------------------------------------------------|
+|  1   |     2     | GAP_AD_TYPE_FLAGS                | 0x06 (BR/EDR Not Supported, LE Gerneral Discoverable) |
+|  2   |     18    | GAP_AD_TYPE_SERVICE_128_BIT_DATA | <DCI Sevice UUID, Number of Resets>                   |
+
+![advertising data](assets/advertising_data.PNG)
+
+***Scan Response Data***
+| AD # | AD Length | AD Type                          | AD Data |
+|------|-----------|----------------------------------|---------|
+|  1   |     4     | GAP_AD_TYPE_COMPLETE_NAME        | "DCI"   |
+
+![scan response data](assets/scan_rsp_data.PNG)
```

Comment:
Github not displaying image as README uses uppercase extension (.PNG) but file uses a lowercase extension (.png)

Reply:
Fixed png extension